app/views/admin/sitewide_settings/index.html.erb

```diff
@@ -1,33 +1,39 @@
 <% content_for :page_title, "Sitewide settings" %>
 <% content_for :title, "Sitewide settings" %>
-
-<% unless @sitewide_settings.present? %>
-  <p class="govuk-body">No sitewide settings available to configure.</p>
-<% end %>
-
-<%= render "govuk_publishing_components/components/table", {
-  head: [
-    {
-      text: "Name"
-    },
-    {
-      text: "On/Off"
-    },
-    {
-      text: "Edit"
-    }
-  ],
-  rows: @sitewide_settings.map do |sitewide_setting|
-    [
-      {
-        text: link_to(sitewide_setting.name, edit_admin_sitewide_setting_path(sitewide_setting), title: "View #{sitewide_setting.name}", class: "govuk-link")
-      },
-      {
-        text: sitewide_setting.human_status
-      },
-      {
-        text: link_to("Edit", edit_admin_sitewide_setting_path(sitewide_setting), title: "Edit #{sitewide_setting.name}", class: "govuk-link")
-      },
-    ]
-  end
+<% content_for :title_margin_bottom, 4 %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% unless @sitewide_settings.present? %>
+      <p class="govuk-body"></p>
+      <%= render "components/inset_prompt",{
+        description: "No sitewide settings available to configure."
+      }%>
+    <% end %>
+    <%= render "govuk_publishing_components/components/table", {
+        head: [
+          {
+            text: "Name"
+          },
+          {
+            text: "On/Off"
+          },
+          {
+            text: "Edit"
+          }
+        ],
+        rows: @sitewide_settings.map do |sitewide_setting|
+          [
+            {
+              text: link_to(sitewide_setting.name, edit_admin_sitewide_setting_path(sitewide_setting), title: "View #{sitewide_setting.name}", class: "govuk-link")
+            },
+            {
+              text: sitewide_setting.human_status
+            },
+            {
+              text: link_to("Edit", edit_admin_sitewide_setting_path(sitewide_setting), title: "Edit #{sitewide_setting.name}", class: "govuk-link")
+            },
+          ]
+        end
 } if @sitewide_settings.present? %>
```

Comment:
can we outdent this please so it's clear it's closing off the table component please